neo4j/datadog_checks/neo4j/neo4j.py

```diff
@@ -1,167 +1,310 @@
-from datadog_checks.base.checks import AgentCheck
-from datadog_checks.base.errors import CheckException
-
-
-class Neo4jCheck(AgentCheck):
```

Comment:
This used to be an `AgentCheck`, but you change it to `PrometheusCheck`. There may be some users using the older versions of this check, but up to you if you want to get rid of the old logic completely.

Reply:
Going to leave it as is.